crates/sui-indexer-alt-framework/src/cluster.rs

```diff
@@ -175,16 +175,17 @@ mod tests {
 
     use diesel::{Insertable, QueryDsl, Queryable};
     use diesel_async::RunQueryDsl;
+    use sui_indexer_alt_framework_store_pg::pg_store::PgStore;
     use sui_synthetic_ingestion::synthetic_ingestion;
     use tempfile::tempdir;
 
     use crate::db::temp::{get_available_port, TempDb};
     use crate::db::Db;
     use crate::pipeline::concurrent::{self, ConcurrentConfig};
     use crate::pipeline::Processor;
-    use crate::store::Store;
     use crate::types::full_checkpoint_content::CheckpointData;
     use crate::FieldCount;
+    use sui_indexer_alt_framework_store_traits::Store;
```

Comment:
You should not have had to change this import.

```suggestion
    use crate::store::Store;
```